config/main.py

```diff
@@ -321,7 +322,11 @@ def load_mgmt_config(filename):
                 expose_value=False, prompt='Reload config from minigraph?')
 def load_minigraph():
     """Reconfigure based on minigraph."""
-    #Stop services before config push
+    # Check /etc/sonic/minigraph.xml file exists
+    if not os.path.isfile(SONIC_MINIGRAPH_PATH):
```

Comment:
>if not os.path.isfile(SONIC_MINIGRAPH_PATH): [](start = 4, length = 44)

I suggest test it by command ```sonic-cfggen -m```

Reply:
Agree here, sonic-cfggen might even do some sanity check here.